cmd/operator/main.go

```diff
@@ -171,7 +171,11 @@ func init() {
 	// specified.
 	flagset.StringVar(&cfg.ReloaderConfig.Image, "prometheus-config-reloader", operator.DefaultPrometheusConfigReloaderImage, "Prometheus config reloader image")
 	flagset.StringVar(&cfg.ReloaderConfig.CPU, "config-reloader-cpu", "100m", "Config Reloader CPU request & limit. Value \"0\" disables it and causes no request/limit to be configured.")
+	flagset.StringVar(&cfg.ReloaderConfig.CPURequest, "config-reloader-cpu-req", "", "Config Reloader CPU request. Value \"0\" disables it and causes no request to be configured.")
```

Comment:
We would also want to change the memory request flag to be "config-reloader-memory-request".

I chose to go with req instead of request just to keep the flag length down. Dunno how much concern there is around that. If none, then i'll change the flags to "config-reloader-cpu-request" and "config-reloader-memory-request".

Reply:
There is no concern, request is better for both agreed.

Reply:
Sounds good, changed both